internal/exec/workflow_utils.go

```diff
@@ -32,15 +33,16 @@ func ExecuteWorkflow(
 		return fmt.Errorf("workflow '%s' does not have any steps defined", workflow)
 	}
 
-	logFunc := u.LogDebug
+	// Logging: use log.Info for dryRun, log.Debug otherwise
+	logFn := log.Debug
 	if dryRun {
-		logFunc = u.LogInfo
+		logFn = log.Info
```

Comment:
I think it's really wierd we change log levels based on dry run. We should either log it or not, and the dryRun shouldn't change the log levels.

Reply:
fine by me to change them all to a fixed log level. I'll update